source/multigrid/mg_transfer_matrix_free.cc

```diff
@@ -719,19 +719,21 @@ namespace
                              VectorizedArray<Number> *data)
   {
     Assert(degree > 0, ExcNotImplemented());
-    const unsigned int loop_length = 2*degree+1;
+    const int loop_length = 2*degree+1;
```

Comment:
No, because then I compare signed with unsigned integers in the loops below, which the compiler will warn about. (The problem the comparison `i<loop_length-1` for `loop_length=1`.)